src/core/views.py

```diff
@@ -3189,26 +3189,26 @@ def get_form_kwargs(self):
         }
 
         if self.decision == 'accept':
```

Comment:
Suggestion: 
The nice thing about the functions you've created is that this `if/else` can now be cleaned up quite a bit and simplified to something like
```
if self.decision == 'accept':
    setting_name = 'proposal_requested_reviewer_accept'
else:
    setting_name = 'proposal_requested_reviewer_decline'

email_body = email.get_email_body(
    request=self.request,
    setting_name=setting_name,
    context=email_context,
)
subject_setting = setting_name + '_subject'
email_subject = email.get_email_subject(
    request=self.request,
    setting_name=subject_setting,
    context=email_context,
)
```